css/styles.css

```diff
@@ -537,6 +533,212 @@
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    justify-content: start;
+    box-sizing: border-box;
+    gap: 0.4rem;
+    border: 1px solid #dfe1e6;
+    box-shadow: 0 2rem 2rem rgba(37, 47, 137, 0.08);
+    overflow-y: scroll;
+  }
+
+  .popup-title {
+    width: 100%;
+    height: 2rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    order: 0;
+  }
+
+  .popup-pr-title {
+    position: relative;
+    margin: 0;
+    font-family: "Poppins", sans-serif;
+    font-weight: 700;
+    font-size: 1.6rem;
+    line-height: 44px;
+    color: #172b4d;
+    order: 0;
+  }
+
+  .popup-close-btn {
+    order: 1;
+  }
+
+  .popup-toggle {
+    position: relative;
+    font-size: 1.5rem;
+    height: 1.2rem;
+    width: 1.5rem;
+    border: 0;
+    margin-right: 0.3rem;
+    margin-top: 0.5rem;
+    color: #000;
+    background: #fff;
+  }
+
+  ul.popup-pr-type {
+    order: 1;
+    position: relative;
+    width: auto;
+    margin: 0;
+    padding: 0;
+    height: 2rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: start;
+    gap: 24px;
+  }
+
+  .popup-pr-photo {
+    position: relative;
+    width: 100%;
+    height: auto;
+    order: 2;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .popup-portfolio-img {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    box-sizing: border-box;
+    height: auto;
+    scale: 1;
+    transition: scale 1s ease-in-out;
+  }
+
+  .popup-portfolio-img:hover {
+    scale: 1.05;
+  }
+
+  .popup-details,
+  .popup-tech-info {
+    position: relative;
+    margin: 0 auto;
+    padding: 0;
+    width: 100%;
+    height: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    justify-content: start;
+    gap: 0.6rem;
+  }
+
+  .popup-details {
+    order: 3;
+  }
+
+  .popup-pr-desc {
+    position: relative;
+    height: auto;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    text-align: justify;
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    align-self: start;
+    color: #344563;
+    order: 0;
+  }
+
+  .popup-tech-info {
+    order: 1;
+  }
+
+  ul.popup-pr-techs {
+    position: relative;
+    height: 1.5rem;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: start;
+    align-items: start;
+    gap: 6px;
+    width: auto;
+    order: 0;
+  }
+
+  .popup-divider {
+    position: relative;
+    margin: 0.12rem auto;
+    width: 100%;
+    border: 1px solid #ebecf0;
+    order: 1;
+  }
+
+  .popup-action-btns {
+    position: relative;
+    padding: 0;
+    margin: 0;
+    height: auto;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    order: 2;
+  }
```

Comment:
- [ ] Nice work implementing the popup windows :+1: :sunglasses: However the gap between the buttons on the mobile version is larger compared to the Figma design. 

### Your Design :point_down: 
![Screenshot from 2023-04-20 18-36-46](https://user-images.githubusercontent.com/91142432/233458092-edbacad2-1376-4297-ac39-d7cc2ba30aae.png)

### Figma Design :point_down: 
![Screenshot from 2023-04-19 13-30-07](https://user-images.githubusercontent.com/91142432/233458170-f39a47ca-bffc-421a-ba96-5f84705f2aa9.png)

Reply:
Guess, I missed that. Thanks.